app/api/temp_message/messages/create.rb

```diff
@@ -0,0 +1,28 @@
+module TempMessage
+  module Messages
+    class Create < Grape::API
+      desc 'Creates a spline that can be reticulated.'
+      params do
+        requires :message, type: Hash do
+          requires :content, type: String, desc: 'Content of the message.'
+          requires :expiration_limit,
+                   type: Integer,
+                   desc: 'Amount, in terms of the expiration type, that the message will be available.'
+          requires :expiration_type,
+                   type: String,
+                   desc: 'Type of expiration.',
+                   values: ::Message.expiration_types.keys
+        end
+      end
+
+      post '', jbuilder: 'messages/create' do
+        message_params = declared(params, include_missing: false)[:message]
```

Comment:
I think, this is like `params.require(:message).permit(:content, :expiration, :expiration_type)` with ApplicationController